examples/with-tailwind/packages/ui/src/client/index.tsx

```diff
@@ -0,0 +1,3 @@
+// components
+export * from "./counter";
+export * from "./counters";
```

Comment:
We need to do our best to move off of barrel files since they're a suboptimal practice. Any ideas?

Reply:
You don’t need to have them. I just have them in case people want to `import { Card } from “ui”`. The example in the web project already does stuff like `import { Counters } from `“ui/client/counters”`.

Reply:
I think `<Card/>` is still imported from `ui` but I will update it to import from `ui/card`.

Reply:
`docs` workspace updated to `import { Card } from 'ui/card'` in https://github.com/vercel/turbo/pull/6307/commits/f9db4e4b57ff46520e9821121adf4758f78456c3, the other examples already load from non barrel files.

@anthonyshew do you want to eliminate the barrel files (index files) all together or keep them around to make both import option open?